front/lib/api/files/text_extraction.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  isTextExtractionSupportedContentType,
+  pagePrefixesPerMimeType,
+  TextExtraction,
+} from "@dust-tt/types";
+import { buffer } from "stream/consumers";
+
+import config from "@app/lib/api/config";
+import type { Authenticator } from "@app/lib/auth";
+import type { FileResource } from "@app/lib/resources/file_resource";
+import logger from "@app/logger/logger";
+
+export const extractTextFromFile = async (
```

Comment:
Can this returns a `Result`? to avoid throwing?

Reply:
There was a comment saying that using in pipeline required throwing.
As discussed IRL, will do another PR to bring back pipeline, therefore need the throwing, therefore will not do it right now.